scripts/jenkins/init-node.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# Install nvm for this shell
+source ~/.nvm/nvm.sh
+
+# Use version of node.js found in .nvmrc
+nvm install
+
+npm install -g yarn
```

Comment:
We should install a version matching what we have in package.json's `engines.yarn` field.